nextflow.config

```diff
@@ -78,7 +79,7 @@ params {
     // References
     cmgg_config_base           = "/conf/"
     igenomes_base              = null //'s3://ngi-igenomes/igenomes'
-    igenomes_ignore            = true
+    igenomes_ignore            = false
```

Comment:
One or the other should be true here (preferably `genomes`) since that simplifies config in the production env

Reply:
Good point!